format/FormatUtils.js

```diff
@@ -43,3 +44,18 @@ export function asElementDeprecationWarning(opts) {
         msg: 'Formatters return elements by default. You can use `asHtml` to return a HTML string'
     });
 }
+
+/**
+ * Used to ensure grid renderers cannot bring down an application
+ */
+export function makeRendererSafe(fn) {
```

Comment:
let's put this in the `/cmp/grid/impl`.  its going to be pretty specific, and no need to reveal it as a public API

Reply:
+ a bit curious about whether the spread and destructure adds any overhead (versus explicit arguments) 
+ might be helpful to include the name of the column (or 'GROUPROW') in the error message
+ we could consider whether to also wrap the export function, but I think better not to